learning/tour-of-beam/backend/internal/storage/image/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Version. Can change in build progress
+ARG GCLOUD_SDK_VERSION=397.0.0-emulators
+
+# Use google cloud sdk
+FROM google/cloud-sdk:$GCLOUD_SDK_VERSION
+
+# Volume to persist Datastore data
+VOLUME /opt/data
+
+# RUN mkdir -p /opt/data/WEB-INF
+COPY index.yaml /opt/data/WEB-INF/index.yaml
```

Comment:
Please, don't repeat mistakes from playground service :)
Please minimize the number of layers according to best practices
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers